back/engines/commercial/machine_translations/spec/services/machine_translations/machine_translation_service_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe MachineTranslations::MachineTranslationService do
+  let(:service) { described_class.new }
+
+  describe 'translate' do
```

Comment:
I think there are cases missing from this test. The `translate` method has more complex behaviour than returning the corrected translated HTML. For instance, exceptions and the case of returning an empty string are not covered.

Reply:
Indeed, I considered it out of scope for this PR, but fine for me to add more tests

Reply:
Perhaps it's also a good opportunity to stub the translations in the acceptance tests.